Xamarin.Forms.Platform.Android/Renderers/EditorRenderer.cs

```diff
@@ -165,12 +167,23 @@ void UpdateInputType()
 			var keyboard = model.Keyboard;
 
 			edit.InputType = keyboard.ToInputType() | InputTypes.TextFlagMultiLine;
-			if (!(keyboard is Internals.CustomKeyboard) && model.IsSet(InputView.IsSpellCheckEnabledProperty))
+			if (!(keyboard is Internals.CustomKeyboard))
 			{
-				if ((edit.InputType & InputTypes.TextFlagNoSuggestions) != InputTypes.TextFlagNoSuggestions)
+				if (model.IsSet(InputView.IsSpellCheckEnabledProperty))
```

Comment:
Can we simplify these next few `if` statements? I think the end result is that `TextFlagNoSuggestions` is always set to 'on' if `IsSpellCheckEnabled` or `IsTextPredictionEnabled` are true, and remains unchanged otherwise?

Reply:
looking into that.